Mathlib/RepresentationTheory/FdRep.lean

```diff
@@ -197,3 +198,45 @@ theorem dualTensorIsoLinHom_hom_hom : (dualTensorIsoLinHom ρV W).hom.hom = dual
 #align fdRep.dual_tensor_iso_lin_hom_hom_hom FdRep.dualTensorIsoLinHom_hom_hom
 
 end FdRep
+
+noncomputable section equivFullSubcategory
+
+namespace FdRep
+
+variable {k G : Type u} [Field k] [Monoid G] {V W : FdRep k G}
+
+/- TODO: Unify how to phrase lemmas/theorems - Use V? V.toRep? Or V.ρ.asModule? -/
+instance toRep_finiteDimensional : FiniteDimensional k V.toRep :=
+  FGModuleCat.instFiniteCarrier k _
+
+/-- Lift `FdRep` morphisms to `Rep` morphisms via the forgetful functor. -/
+def lift_hom (f : V ⟶ W) : V.toRep ⟶ W.toRep :=
+  (forget₂ _ _).map f
```

Comment:
It seems to me this is unused. Could be safely removed?

Reply:
I thought it would be helpful in general, since there is `toRep` on objects there should also be a map for morphisms. But I can remove yes

Reply:
For now, I think we should remove it. (Also, it does not follow mathlib's naming conventions.)